src/modules/editFunction.js

```diff
@@ -0,0 +1,17 @@
+export default function editTask(desc, liElement, div1, div2, obj, array) {
+  desc.onfocus = () => {
```

Comment:
- [ ] Please have your scope variables be fully understandable, as incomplete words are confusing/disrupting and can be considered as test/unneeded code.